### PR TITLE
Validation add descriptions

### PR DESCRIPTION
**Pull request by chrisms86**

### CODE REVIEW

File: iis/extensions/smooth-streaming-client/advancedloggingplugin-manifestlogurls-property-microsoft-web-media-diagnostics.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: AdvancedLoggingPlugin.ManifestLogUrls Property (Microsoft.Web.Media.Diagnostics)
+description: This article contains syntax, version information, and permissions for the ManifestLogUrls Property.
```
Comment (MeeraDi) [inline iis/extensions/smooth-streaming-client/advancedloggingplugin-manifestlogurls-property-microsoft-web-media-diagnostics.md:3]: Based on the previous, shouldn't this be "for the AdvancedLoggingPlugin.ManifestLogUrls property"?

File: iis/extensions/smooth-streaming-client/trackinfo-stream-property-microsoft-web-media-smoothstreaming.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: TrackInfo.Stream Property (Microsoft.Web.Media.SmoothStreaming)
+description: This article contains syntax, version information, and permissions, and reference materials for the Stream Property.
```
Comment (MeeraDi) [inline iis/extensions/smooth-streaming-client/trackinfo-stream-property-microsoft-web-media-smoothstreaming.md:3]: ...reference materials for the TrackInfo.Stream property.

**Comment by chrisms86** [comment]
**Global effort to fix build validation errors**
@Rick-Anderson @mcleblanc 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Review by Rick-Anderson — APPROVED** [review]

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]
Please review warnings

**Pull request merged**